src/Compilers/CSharp/Portable/Symbols/Source/SourceClonedParameterSymbol.cs

```diff
@@ -56,6 +56,8 @@ internal override bool IsMetadataOptional
             }
         }
 
+        internal override BoundExpression ExplicitDefaultExpression => _originalParam.ExplicitDefaultExpression;
```

Comment:
Can you explain why null is the more appropriate result, for both this case and WrappedParameterSymbol? I'm not sure I understand the reasoning.